templates/CRM/Admin/Form/Setting/Search.tpl

```diff
@@ -68,10 +68,17 @@
             <td>{$form.smartGroupCacheTimeout.html}<br />
                 <span class="description">{ts}The number of minutes to cache smart group contacts. We strongly recommend that this value be greater than zero, since a value of zero means no caching at all. If your contact data changes frequently, you should set this value to at least 5 minutes.{/ts}</span></td>
         </tr>
+        <tr class="crm-search-setting-form-block-default_quicksearch_option">
```

Comment:
Ideally we'd switch this form to be just using the generic form CRM/Form/basicForm.tpl  - see the Multisite, Mailing & Member settings forms for an example. We have been trying to migrate forms as we touch them (as preliminary tidy up) - I'm not sure if there are any blockers to that in this case because sometimes there are tricky bits of logic to work through)

It's also possible to only switch some fields but specifying the subset of fields that 'can just be switched' in an array and then including CRM/Form/basicFormFields.tpl